packages/nuxt/src/app/composables/cookie.ts

```diff
@@ -32,10 +32,11 @@ export function useCookie<T = string | null | undefined> (name: string, _opts?:
   const cookies = readRawCookies(opts) || {}
 
   let delay: number | undefined
-  if (opts.maxAge) {
+
+  if (opts.maxAge !== undefined) {
     delay = opts.maxAge * 1000 // convert to ms for setTimeout
   } else if (opts.expires) {
-    // getTime() already return time in ms
+    // getTime() already returns time in ms
     delay = opts.expires.getTime() - Date.now()
   }
   // use customRef if on client side otherwise use basic ref
```

Comment:
what about just checking that `delay >= 1` on L43 so we don't use a `cookieRef` in that case?